graql/language/match.feature

```diff
@@ -1450,9 +1450,9 @@ Feature: Graql Match Query
     Given graql insert
       """
       insert
-      $x isa person, has name "Zoe", has age 21, has graduation-date 2020-06-01, has ref 0;
+      $x isa person, has name "Zoe", has age 21, has graduation-date 2021-06-01, has ref 0;
```

Comment:
let's revert it

Reply:
@flyingsilverfin @lolski addressed in https://github.com/graknlabs/behaviour/pull/136/commits/98f340f72238de28ce14dc2f629c82b1b1fc6242